src/frontend/web_application/src/scenes/UserSecurity/presenter.jsx

```diff
@@ -11,40 +12,76 @@ class UserSecurity extends Component {
   static propTypes = {
     __: PropTypes.func.isRequired,
     requestUser: PropTypes.func.isRequired,
-    user: PropTypes.shape({}),
+    user: PropTypes.shape({}).isRequired,
+    notifySuccess: PropTypes.func.isRequired,
+    notifyError: PropTypes.func.isRequired,
   };
 
-  static defaultProps = {
-    user: undefined,
+  state = {
+    updated: false,
   };
 
   componentDidMount() {
     this.props.requestUser();
   }
 
+  handleSubmit = (data) => {
```

Comment:
we will have to move this to redux for consistency & separation of concerns.

For now it sounds to work so ok